lib/utils.js

```diff
@@ -386,6 +448,7 @@ function getActionUrls (config, /* istanbul ignore next */ isRemoteDev = false,
 }
 
 function removeProtocolFromURL (url) {
+  // todo: explain yourself!
```

Comment:
should this be explained (or deleted)? :D

Reply:
just making a note, it has nothing to do with this pr.
I personally expect regex' to be annotated precisely, this does not seem to do what the name of the function says.